README.md

```diff
@@ -1 +1,85 @@
-# node-arguments
\ No newline at end of file
+# Node Arguments
+
+## Overview
+
+This lesson will cover the Node command-line arguments and how to access them.
+
+## Objectives
+
+1. Describe how the CLI argument input to Node script works
+1. Describe an argument array
+2. Describe first arguments
+
+## CLI Arguments
+
+How do we pass data to Node? `node` is just a command-line program and we can pass arguments to it. For example, we can pass two numbers to a program (file `sum.js`) that calculates the sum of them:
+
+```
+$ node sum.js 10 47
+```
+
+The result will be 57. We can pass numbers or strings. This approach is useful when you're running scripts which require small enough input that you can fit in in the command line, or whin building command-line tools with Node. Another use caes is when you have a complex program that needs to be launched in different modes, e.g., development mode with more verbose logging.
+
+## Accessing Arguments
+
+You've learned how to pass arguments, but how can you access them when you write your own program instead of using existing programs like sum.js? It's straightforward. All you need is to access `process.argv`. It's an array so all the Array methods available. 
+
+Consider the sum.js program. I'm using `shift` to get rid of the first two elements and then `forEach()` to iterate through indefinite number of arguments. I convert each argument to a number and add them up to the `sum` variable:
```

Comment:
Get rid of "I" - "First, we use `shift` to get rid of the first two elements`....We convert each argument to a numer..."

also, why are we getting rid of the first two elements? what are they that we don't want them? why are the numbers the 3rd and 4th elements in the array?